### ISSUE TITLE
Adding Protonmail crashes Franz

### ISSUE THREAD

**Issue by squirrelboot** [meetfranz/franz#1428]
**Describe the bug**
Protonmail is not working, and causes the program to crash. 

**To Reproduce**
1. Add service for Protonmail

Without even getting a chance to log in, it crashed Franz. Upon re-loading, it was having issues working properly (CPU suddenly went crazy busy), and crashed. Next time I restarted it, and disabled the service, then removed it. Tried next to add it through custom website, also caused issues/performance issues. 

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Brave, (chromium)
- Browser: 74.0.3729.131
 - Version Franz: 5.1.0

**Comment by closingin** [comment]
Hi,

I managed to generate a service crash, but I don't know how I did it, as it worked 9 times outta 10 on Mojave 10.14.4. This bug requires investigation.

Normally I would advise you to open your issue in the [appropriate repository](https://github.com/lukas-reineke/franz-recipe-proton-mail), but after reviewing the code of the service, I haven't found anything that could cause a crash. I think something else triggered it.
